build.gradle

```diff
@@ -1,17 +1,16 @@
 buildscript {
-    ext {
-        springBootVersion = "1.2.1.RELEASE"
-    }
     repositories {
         mavenLocal()
         jcenter()
         mavenCentral()
         maven { url "http://repo.spring.io/libs-snapshot" }
+        maven { url 'https://oss.sonatype.org/content/groups/public/' }

```

Comment:
Yeah it was for accurest - now it's in maven central so this can be also removed. The snapshot thing must have been there in the GUI branch so I just left it as it is.